sources/models/defaults.toml

```diff
@@ -52,6 +52,24 @@ template-path = "/usr/share/templates/updog-toml"
 affected-services = ["updog"]
 seed.setting-generator = "bork seed"
 
+# Healthdog
+
+[settings.healthdog]
+# the URL to which anonymous health metrics will be sent
+metrics-url = "https://metrics.bottlerocket.aws/metrics"
+# whether or not health metrics will be sent. set to false to opt-out
+send-metrics = true
+# the list of services that are checked to determine if a host is healthy
+service-health = ["apiserver"]
```

Comment:
"service-health" isn't very clear to me. When I first saw it in the model, I assumed you were using it as a storage location for service health check results, and I was confused. How about "check-services" or similar?

Reply:
Sure, or just "services"?

Reply:
Addressed in [f18d841](https://github.com/bottlerocket-os/bottlerocket/compare/ec0abbf943fe3806294f53fd4e118be3d1f529fc..f18d841ee1c2b35d5f09136ea3f7136e2a4d65ae)